src/time.cr

```diff
@@ -331,9 +331,10 @@ struct Time
     now(Location::UTC)
   end
 
-  # Returns a copy of `self` with time-of-day components (hour, minute, ...) set to zero.
-  def date : Time
-    Time.new(year, month, day, location: location)
+  # Returns a `Tuple` with `year`, `month` and `day`.
+  def date : Tuple(Int32, Int32, Int32)
```

Comment:
WDYT about calling it date_parts and using a named tuple for the result?

Since `date` is `at_beginning_of_day` it's maybe fine to drop date I guess...

Reply:
I don't like using a named tuple for this. The fields are inherently ordered, so there is no need to use named accessors. A tuple can be easily decomposed as `year, month, date = time.date`.